test/integration/targets/cnos_vlan/tasks/cli.yaml

```diff
@@ -0,0 +1,22 @@
+---
+- name: collect all cli test cases
+  find:
+    paths: "{{ role_path }}/tests/cli"
+    patterns: "{{ testcase }}.yaml"
+  register: test_cases
+  delegate_to: localhost
+
+- name: set test_items
+  set_fact: test_items="{{ test_cases.files | map(attribute='path') | list }}"
+
+- name: run test cases (connection=network_cli)
+  include: "{{ test_case_to_run }}"
+  with_items: "{{ test_items }}"
+  loop_control:
+    loop_var: test_case_to_run
+
+#- name: run test case (connection=local)
```

Comment:
If you support local, I'd prefer to see this run on local if you don't have a catch-all sanity test suite

Reply:
Ok. Missed to remove commenting after testing.